suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -407,13 +407,12 @@ export const startDiscoveryThunk = createThunk(
                 await dispatch(initMetadata(false));
             }
 
-            dispatch({
-                type: startDiscovery.type,
```

Comment:
this is some old way of dispatching actions and it was hard to find the usages when used this way, lets use actionCreator instead